# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/iproute2/retypedef.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./include-glibc/glibc-bugs.h.orig	2003-11-01 21:41:57.000000000 -0300
+++ ./include-glibc/glibc-bugs.h	2003-11-01 21:48:48.000000000 -0300
@@ -12,8 +12,8 @@
 
 #include <linux/types.h>
 
-typedef __u16 in_port_t;
-typedef __u32 in_addr_t;
+/* typedef __u16 in_port_t; */
+/* typedef __u32 in_addr_t; */
 
 #endif
 
